src/uct/ib/base/ib_md.c

```diff
@@ -1320,6 +1325,12 @@ uct_ib_md_open(const char *md_name, const uct_md_config_t *uct_md_config, uct_md
         md->config.odp.max_size = uct_ib_device_odp_max_size(&md->dev);
     }
 
+    if (inet_pton(AF_INET6, md_config->ext.subnet_prefix, &gid) == 1) {
```

Comment:
minor: maybe it is worth to distinguish 2 cases: empty filter and invalid filter?
If it is the latter - then print some warning/message

Reply:
also, i see user has to provide full 128 bits of IPv6 but only first 64 bits are significant
maybe need to parse it as 64-bit hex value